modules/backend/patients/patient/payments/Payments.hpp

```diff
@@ -7,6 +7,7 @@ namespace patient
 
 struct Payments
 {
+    // TODO: consider type: float or double maybe?
     std::uint8_t paid_;
     std::uint8_t planned_;
     std::uint8_t overdue_;
```

Comment:
From my pov we could use integer types and simply move two digits. 
This way we will omit possible issues with floating types.

Reply:
maybe. I don't understand exactly. Oh I see: for example 100 for 1 zł for example. Good idea.
But any way (and especially if so) we must provide bigger type. If we stick with uint8_t we are constrained to 256.
If we decide to apply 100 for 1 rule this will mean maximium of 2,56 (currency) in our code.

Reply:
Yes, we should go for uint32_t probably. It should be more than enough.
But that's for the future